andela/thor-ah-frontend
# #159987652 highlight and comment on any text

#### What does this PR do?
Adds ability to highlight and comment on any part of an article.

#### Description of Task to be completed?
- ability to highlight any text on article
- ability to comment and comment associated with highlighted text
- Design of interface for comment with highlighted text
- link highlighted comment to to target on click
#### How should this be manually tested?
- Clone the repo
- cd into the ‘thor-ah-frontend’
- run 'npm install'
- run 'npm start’
- navigate to an article page
- highlight any text and click comment icon when it appears
- type your comment in the comment box provided and click submit
- scroll to comments and click on comment with highlighted text to see where in the article it is associated.

#### What are the relevant pivotal tracker stories?
[#159987652](https://www.pivotaltracker.com/story/show/159987652)
